acceptance/ui/step-definitions/user-auth-ui.step-definition.ts

```diff
@@ -0,0 +1,55 @@
+import { expect } from '@playwright/test';
+import { binding, given, then, when } from 'cucumber-tsflow';
+import { BaseStepDefinition } from '../../../src/execution-flow/base.step-definition';
+import { SessionContext } from '../../../src/execution-flow/session.context';
+import { BasePage, HomePage, LoginPage } from '../../../src/pages/ui';
+
+@binding([SessionContext])
+export class UserAuthStepDefinition extends BaseStepDefinition {
+  public constructor(protected sessionContext: SessionContext) {
+    super(sessionContext);
+  }
+
+  @given(/I am on the UI login page/, 'UI', 100000)
+  public async givenLoginPageIsOpened(): Promise<void> {
+    const loginPage = new LoginPage(this.sessionContext.page ?? (() => { throw new Error('page is null'); })());
+    await loginPage.goto(process.env.HOBBYFARM_UI_URL as string, '/login?returnUrl=%2Fapp%2Fhome');
```

Comment:
For the login, you should reproduce the `UiFlow` to be able to manage the changes between version.

```ts
let homePage = await UiFlow.login(this.sessionContext.page ?? (() => { throw new Error('page is null'); })());
```

Reply:
I'm making the code for Admin UI scenario on #18